docs/queries/filters.md

```diff
@@ -21,16 +21,30 @@ parent: Queries
 
 ## Searching for dates
 
-### Specific dates
+In Tasks query blocks, Tasks allows a lot of flexibility in the interpreting of dates.
```

Comment:
how about a shorter version to avoid repeating Tasks?

Tasks allows a lot of flexibility in the dates of query blocks.

Reply:
> Tasks allows a lot of flexibility in the dates of query blocks.

Nice.

I opted for:

> Tasks allows a lot of flexibility in the dates inside query blocks.